pokemon/src/main/java/gamarket/Move.java

```diff
@@ -0,0 +1,135 @@
+package gamarket;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
+/**
+ * Represents all the moves in the Pokemon universe. 
+ * A Pokemon will be able to hold at most 4 moves in its move-set. 
+ */
+public class MoveCollection {
```

Comment:
Mismatched file name? Move.java =/= public class MoveCollection

Reply:
Did a pull request to show the updated Mova.java code.